ui/library/test/context/ScrollContext.test.tsx

```diff
@@ -27,9 +27,10 @@ describe('<ScrollContext/>', () => {
 
   let _setScrollRoot: (root: Nullable<Element>) => any;
   let _resetScrollObservers: () => any;
-  let _scrollToPage: (opts: { pageNumber?: number; pageIndex?: number }) => void;
   let _isOutlineTargetVisible: (dest: NodeDestination) => boolean;
   let _isPageVisible: (opts: { pageNumber?: number; pageIndex?: number }) => boolean;
+  let _scrollToPage: (opts: { pageNumber?: number; pageIndex?: number }) => void;
```

Comment:
I remember you've created a type for `{ pageNumber?: number; pageIndex?: number }`, consider using the type here

Reply:
✅